pkg/runtime/runtime.go

```diff
@@ -1125,10 +1128,12 @@ func (a *DaprRuntime) initSecretStores() error {
 		})
 		if err != nil {
 			log.Warnf("failed to init state store %s named %s: %s", c.Spec.Type, c.ObjectMeta.Name, err)
+			diag.DefaultMonitoring.ComponentInitFailed(c.Spec.Type, "Init")
 			continue
 		}
 
 		a.secretStores[c.ObjectMeta.Name] = secretStore
+		diag.DefaultMonitoring.ComponentInitialized(c.Spec.Type)
```

Comment:
This is component created, no?

Reply:
I need your input here. what I thought is that it basically calls `Init` method in component, so this is initialization of component.